gateway/ln-gateway/Cargo.toml

```diff
@@ -79,6 +79,8 @@ fedimint-dummy-common = { path = "../../modules/fedimint-dummy-common" }
 fedimint-ln-client = { path = "../../modules/fedimint-ln-client" }
 fedimint-ln-server = { path = "../../modules/fedimint-ln-server" }
 fedimint-ln-common = { path = "../../modules/fedimint-ln-common" }
+fedimint-mint-client = { path = "../../modules/fedimint-mint-client" }
+fedimint-wallet-client = { path = "../../modules/fedimint-wallet-client" }
```

Comment:
Ideally we should refactor the tests into a separate crate from `gatewayd`. That would allow us to remove these dependencies for the test crate.

Reply:
Let's do this in a follow-up

Reply:
Unfortunately also needed to add the dummy client here as well. Refactoring to allow `gatewayd` to take in module gens from the outside is too much for this PR.

Reply:
Related issue that should ideally be fixed at the same time: https://github.com/fedimint/fedimint/issues/2650